seeds/03_events_seed.js

```diff
@@ -8,23 +8,23 @@ exports.seed = function (knex) {
         {
           id: 1,
           category_id: 2,
-          user_id: 1,
+          user_id: 2,
```

Comment:
For these, we want a variety of users, so since this is a PR, you can change them back to the id's that were there :)